src/Uno.UI.Runtime.Skia.X11/X11_Bindings/X11Helper.cs

```diff
@@ -334,6 +371,59 @@ private unsafe static void SetMotifWMHints(X11Window x11Window, bool on, IntPtr?
 	[LibraryImport(libX11)]
 	public static partial int XHeightOfScreen(IntPtr screen);
 
+	[LibraryImport(libX11)]
+	public static partial IntPtr XCreateRegion();
+
+	[LibraryImport(libX11)]
+	public static partial int XDestroyRegion(IntPtr region);
+
+	[LibraryImport(libX11)]
+	public unsafe static partial int XUnionRectWithRegion(
+		XRectangle* rectangle,
+		IntPtr src_region,
+		IntPtr dest_region_return
+	);
+
+	public unsafe static IntPtr CreateRegion(short x, short y, short w, short h)
+	{
+		IntPtr region = XCreateRegion();
+		XRectangle rectangle;
+		rectangle.X = x;
+		rectangle.Y = y;
+		rectangle.W = w;
+		rectangle.H = h;
+		var _ = XUnionRectWithRegion(&rectangle, region, region);
+
+		return region;
+	}
+
+	[DllImport(libX11Randr)]
```

Comment:
```suggestion
	[LibraryImport(libX11Randr)]
```

Reply:
DllImport handles string and bool marshalling automatically, so I'm leaving a couple of imports as DllImports.

Reply:
Let's indicate that as a comment